packages/cli/api-importers/openapi/openapi-v2-importer/src/utils/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface GroupNameAndLocation {
```

Comment:
I'm not sure what other types we plan to add here, but we often create separate files for exported types like this. That way it's far easier for developers to find the resource they're looking for. 

Long way of saying - it might be worth making a `GroupNameAndLocation.ts` for this.